scripts/ock-once-swtok.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+PIN=${OCK_PIN:-"12345678"}
+
+if id -u | grep -qw 0 && groups | grep -qwv pkcs11 ; then
+    echo "missing pkcs11 group membership"
+    exit 77
+fi
+
+if ! command -v pkcsslotd > /dev/null; then
+    echo "pkcsslotd is required"
+    exit 77
+fi
+
+for PREFIX in "/usr/local/" "/" "NA"; do
+    if [ "$PREFIX" = "NA" ]; then
+        echo "No ock configuration found or ock-swtok not configured"
+        exit 77
+    fi
+
+    OCK_CONF=${PREFIX}etc/opencryptoki/opencryptoki.conf
+    SLOT=$(grep -B2 "^stdll = libpkcs11_sw.so$" ${OCK_CONF} 2> /dev/null | grep -Po "slot \K[^\s]+")
+    test -n "${SLOT}" && break
+done
+
+if [ $(id -u) != "0" ]; then
```

Comment:
Just no, no root, if root is required let's drop the PR

Reply:
ok.